docs/api/api_blueprint.apib

```diff
@@ -25594,9 +25594,9 @@ This Group's APIs can be used for adding a particular event to the favourite lis
             }
         }
 
-## Favourite Events Detail [/v1/user-favourite-events/{user_favourite_event_id}]
+## Favourite Events Detail [/v1/user-favourite-events/{event_id}]
 + Parameters
-    + user_favourite_event_id: 1 (integer) - ID of the Favourite Event
+    + event_id: 1 (integer) - ID of the Event in the form of an integer
```

Comment:
Changing name of parameters won't fix anything.

Reply:
@iamareebjamal But it should be changed, right?

Reply:
Yes, your version is more explicit, but if you changed it in hopes that it'll fix the error, then it won't